common/web/types/package.json

```diff
@@ -79,5 +82,6 @@
       "src/schemas/*",
       "test/"
     ]
-  }
+  },
+  "sideEffects": false
```

Comment:
Without this specific line, a _lot_ of the package fails to tree-shake out.  Notably, all of the precompiled schemas would get included, which would cause major filesize bloat.

https://github.com/evanw/esbuild/issues/1435#issuecomment-1334487395

While the linked comment talks about _JS_ imports, rather than explicitly-_TS_ imports in the manner I've set up here... the same concerns appear to apply.  

Perhaps a package used by `common/web/types` doesn't include the `sideEffects` package tag, and that's cascading?  Just a hunch... at the moment.  I do see that there are three external non-dev dependencies that it uses... and one of them has its own dependencies.  (None of the three are marked with `sideEffects: false`, either - but doing this isn't enough.)

I've tried investigating this... but without much luck.  All I can say is that something... seems "off" with any attempt to treeshake `common/web/types` without this package's flag.

The most relevant line of thought I can think of for now:  the schema validators are spit out as `.mjs` files, not TS, making that comment above more relevant.  That said, if I locally comment out all Web-ignored exports from `common/web/types/src/main.ts`, then re-enable the `export * as util from './util/util.js'`, that export doesn't fully tree-shake out, and it doesn't refer to the schemas, so... yeah.

Not sure if it's relevant, but I do see a mild difference with `"sideEffects": false` for the gesture-recognizer - the subfolder `index.ts` files - for a common export interface - get tree-shaken out with the flag, but do not when the flag is missing.  For the gesture-engine... it makes all of 0.3kb difference.